### ISSUE TITLE
Article does not explain how to get the new project templates/features

### ISSUE THREAD

**Issue by astrohart** [MicrosoftDocs/visualstudio-docs#8323]
RE: [https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/extensibility/vsix/get-started/first-extension.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/extensibility/vsix/get-started/first-extension.md)

Hello,

There's an issue(s) with the article described above, actually, this applies to the whole "Visual Studio Community Toolkit" series.

The series needs to be more specific as to how does one obtain the Visual Studio Community Toolkit in the first place.

For example, I tried to follow the instructions in the walkthrough that is linked above.  But as soon as I chose **File**, **New Project**, I ran into a snag.  The walkthrough says to select the `VSIX Project w/Command (Community)` template.

However, I did a search in my instance of VS (2019 Enterprise) and I could not find any templates of that name.  I assume there is a download somewhere to obtain these template, but none of the articles in this series say where that download is, or how to obtain it.

I suggest updating the article to mention how to obtain the Toolkit and/or templates in the first place.

**Comment by WilliamAntonRohm** [comment]
@astrohart -- Brian, thank you for your feedback.  Please consider these resources:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@madskristensen -- Mads, please look into this issue.

**Comment by astrohart** [comment]
@WilliamAntonRohm thanks for sending the links.  I'll check them out when I get a chance.  However, I am unsure I will find the answer at any of those places.  I am also well-familiar with all of these avenues.

**Comment by astrohart** [comment]
@WilliamAntonRohm and @madskristensen I also filed an issue [here](https://github.com/VsixCommunity/Community.VisualStudio.Toolkit/issues/368) and got an answer [here](https://github.com/VsixCommunity/Community.VisualStudio.Toolkit/issues/368#issuecomment-1200451586) pointing me to the [Extensibility Template Pack 2022](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.ExtensibilityItemTemplates2022). 

I also found the Getting Started page and it mentions that there is also a VS2019-compatible version of the extension.

@madskristensen I want to reiterate, I think this article series is wonderful, and I am glad to see you're paying attention to beginners.

May I finish by suggesting?  It'd be a strong recommendation of mine that each article in the series have a sidebar at the top: "Don't have the tools?  Get them here: (link)"
